rpc/cache/service.proto

```diff
@@ -58,3 +59,11 @@ message MissingBlobsResponse {
   bool            missing_artifact = 1;
   repeated string missing_blob_ids = 2;
 }
+
+message GetBlobRequest {
+  string blob_id = 1;
+}
+
+message GetBlobResponse {
+  BlobInfo blob_info = 1;
+}
```

Comment:
What are they for?

Reply:
For new remote cache implementation, I wanted to expose getblob, to support local applylayers. But it is not needed now. Will remove it.